units/crons.go

```diff
@@ -842,18 +842,23 @@ func PopulateHostCreationJobs(env evergreen.Environment, part int) amboy.QueueOp
 			if h.UserHost || h.SpawnOptions.SpawnedByTask {
 				// pass:
 				//    always start spawn hosts asap
-			} else if submitted > 32 {
-				// throttle hosts, so that we're starting very
-				// few hosts on every pass. Hostinit runs very
-				// frequently, lets not start too many all at
-				// once.
-
-				continue
 			} else {
-				// only increment for task hosts, since otherwise
-				// spawn hosts and hosts spawned by tasks could
-				// starve task hosts
-				submitted++
+				config, err := evergreen.GetConfig()
```

Comment:
getting just the hostinit section could save db calls since we don't need all the configuration sections

Reply:
fixed